cv.tex

```diff
@@ -44,92 +50,85 @@
+2019 \href{https://www.giac.org/certifications/web-application-penetration-tester-gwapt/}{GWAPT} (\#6137) \\
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
-% Links
+% Skills
 %------------------------------------------------
 
-\section{Links} 
+\section{Skills}
 
-Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
-LinkedIn:// \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
-YouTube:// \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
-Twitter:// \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
-Quora:// \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}
+\subsection{Programming}
+\href{https://aws.amazon.com/}{AWS} \textbullet{} Python \textbullet{} \textbf{\href{https://docs.panther.com/}{Panther}} \textbullet{} SQL \\
+\location{Working Proficiency:}
+C \textbullet{} \href{https://console.cloud.google.com/}{GCP} \textbullet{} Java \textbullet{} Javascript \textbullet{} PHP \textbullet{} \textbf{\href{https://www.oracle.com/cloud/}{OCI}} \\
+\location{Familiar:}
+Boto3 \textbullet{} JSON \textbullet{} \LaTeX\ \textbullet{} Pandas \\
+PySpark \textbullet{} Selenium \textbullet{} YAML \\
+
+\subsection{Analytical}
+AWS Security \textbullet{} Databricks \textbullet {} Snowflake \\
+Suricata \textbullet{} Splunk\\
+\location{Forensic Proficiency:}
+.eml/.msg \textbullet{} Linux \textbullet{} MacOS\textbullet{} Windows \\
+\location{Logs of Familiarity:}
+AWS CloudTrail \textbullet{} GuardDuty \textbullet{} CrowdStrike Falcon Data Replicator \\
+Github Audit \textbullet{} Okta System \textbullet{} Zscaler \\
 
 \sectionspace % Some whitespace after the section
 
+
 %------------------------------------------------
-% Coursework
+% Awards
 %------------------------------------------------
 
-\section{Coursework}
-
-\subsection{Graduate}
-
-Advanced Machine Learning \\
-Open Source Software Engineering \\
-Advanced Interactive Graphics \\
-Compilers + Practicum \\
-Cloud Computing
+\section{Awards}
+2023 Dropbox Team TACO "Capitan" \\
+2021 Amazon Security ”Bias for Action” \\
+2021 US Patent \textbf{\href{https://patents.google.com/patent/US10904292B1/en}{10904292B1}} Granted \\
+2020 US Patent \textbf{\href{https://patents.google.com/patent/US10740151B1/en}{10740151B1}} Granted \\
+2014 1\textsuperscript{st}/10 \textbf{\href{https://www.nationalccdc.org/}{NCCDC}} National Champions (Team Member) \\
 
 \sectionspace % Some whitespace after the section
 
+%------------------------------------------------
+% Societies
 %------------------------------------------------
 
-\subsection{Undergraduate}
-
-Information Retrieval \\
-Operating Systems \\
-Artificial Intelligence + Practicum \\
-Functional Programming \\
-Computer Graphics + Practicum \\
-{\footnotesize \textit{\textbf{(Research Asst. \& Teaching Asst) }}} \\
-Unix Tools and Scripting
+\section{Societies}
```

Comment:
(JC):
- Should this maybe be Leadership? Societies is strong.
- “Hac@kUCF” —> “Hack@UCF”
- If you’ve got room, a one-liner under each title describing each organization could be helpful.

Reply:
1. unsure.
2. omg so embarassing ty
3. good idea but not sure of how to format this... sleeping on it for now but ty.